docs/source/cpp/compute.rst

```diff
@@ -1498,15 +1483,19 @@ in the respective option classes.
 Structural transforms
 ~~~~~~~~~~~~~~~~~~~~~
 
-+--------------------------+------------+------------------------------------+---------------------+---------+
-| Function name            | Arity      | Input types                        | Output type         | Notes   |
-+==========================+============+====================================+=====================+=========+
-| list_element             | Binary     | List-like (Arg 0), Integral (Arg 1)| List value type     | \(1)    |
-+--------------------------+------------+------------------------------------+---------------------+---------+
-| list_flatten             | Unary      | List-like                          | List value type     | \(2)    |
-+--------------------------+------------+------------------------------------+---------------------+---------+
-| list_parent_indices      | Unary      | List-like                          | Int32 or Int64      | \(3)    |
-+--------------------------+------------+------------------------------------+---------------------+---------+
++---------------------+------------+-------------------------------------+------------------+------------------------------+--------+
+| Function name       | Arity      | Input types                         | Output type      | Options class                | Notes  |
++=====================+============+=====================================+==================+==============================+========+
+| list_element        | Binary     | List-like (Arg 0), Integral (Arg 1) | List value type  |                              | \(1)   |
++---------------------+------------+-------------------------------------+------------------+------------------------------+--------+
+| list_flatten        | Unary      | List-like                           | List value type  |                              | \(2)   |
++---------------------+------------+-------------------------------------+------------------+------------------------------+--------+
+| list_parent_indices | Unary      | List-like                           | Int32 or Int64   |                              | \(3)   |
++---------------------+------------+-------------------------------------+------------------+------------------------------+--------+
+| list_value_length   | Unary      | List-like                           | Int32 or Int64   |                              | \(4)   |
++---------------------+------------+-------------------------------------+------------------+------------------------------+--------+
+| make_struct         | Varargs    | Any                                 | Struct           | :struct:`MakeStructOptions`  | \(5)   |
++---------------------+------------+-------------------------------------+------------------+------------------------------+--------+
```

Comment:
"list_value_length" and "make_struct" are scalar functions, but this is moving them to the category of vector functions. I don't think this is ok.